samples/nrf9160/lwm2m_client/README.rst

```diff
@@ -556,6 +556,102 @@ To update the firmware, complete the following steps:
       Restart the device manually if it has not started automatically.
       The device runs the updated firmware and reconnects to Coiote Device Management server automatically.
 
+Preparing for production
```

Comment:
Instead of adding this to _README.rst_ I would consider creating a separate file for this. _Provisioning.rst_, or something similar.

Reply:
Splitted.